app/src/main/java/com/pvp/app/ui/screen/calendar/MonthlyCalendarScreen.kt

```diff
@@ -72,12 +72,26 @@ fun DayDialog(
         properties = DialogProperties(usePlatformDefaultWidth = false)
     ) {
         Day(
-            name = date.toString(),
-            tasks = tasks
+            "${date.month.toString().capitalizeFirstLetter()} ${date.dayOfMonth}",
+            tasks = tasks,
+            date = date,
+            expandedUponCreation = true
         )
     }
 }
 
+fun String.capitalizeFirstLetter(): String {
```

Comment:
Pretty sure methods (extensions) for this already exist. Also, put such util elsewhere.

- If utility is required to execute something very specific to associated screen from multiple places, put it under that screen's package.
- If utility can be used in various situations (like now, where it is not tied to some context, it just makes String (used everywhere) to start by a capital latter), it should be placed within `app/common/Utils.kt`.
- If utility can be used in various situations, but only for UI, it should be placed within `/app/ui/common/Utils.kt`

Actual file name can be different, if grouped by some logic.